backend/app/libs/storage_helpers/tokenlists.py

```diff
@@ -1,15 +1,63 @@
+import os
```

Comment:
Apart from the slight remarks that I made I believe the code looks good! As the `tokenlists` module gets a bit more complex (and in an effort to have a clean structure in the codebase), the code would better sit in a different top-level module `app.tokenlists`, with a hexagonal(-ish) structure, as done for `treasury` for example. You'd have adapters for:

- storage (retrieve_token_whitelist, store_token_whitelist)
- covalent
- tokenlists json URLs.

I suggest doing this refactoring in a separate PR.